### ISSUE TITLE
TODO: Handle screen recording not trigger listener event on iOS

### ISSUE THREAD

**Issue by gbumps** [gbumps/react-native-screenguard#40]
issue related #38.

**Comment by gbumps** [comment]
Add `UIScreenCapturedDidChangeNotification` event listener

**Comment by Simoon-F** [comment]
concern.

**Comment by gbumps** [comment]
updated: version `0.3.6`. Please check this first before releasing: 

```
    "react-native-screenguard": "git+https://github.com/gbumps/react-native-screenguard.git#release/0.3.6",
```

**Comment by Simoon-F** [comment]
> @Simoon-F @mthinh
> 
> updated at version `0.3.6`. Please check this first before releasing:
> 
> ```
>     "react-native-screenguard": "git+https://github.com/gbumps/react-native-screenguard.git#release/0.3.6",
> ```
cool. 👍

**Comment by Simoon-F** [comment]
run `pod install` prompt version is `0.3.5`. @gbumps 

package.json

```
"react-native-screenguard": "git+https://github.com/gbumps/react-native-screenguard.git#release/0.3.6",
```

pod install

```
Installing react-native-screenguard (0.3.5)
```

**Comment by gbumps** [comment]
Hi, i've pushed new commit updated for the iOS. You can head over to the installed version of `react-native-screenguard` inside the `node_modules` folder, and go to `react-native-screenguard.podspec`, modify the `s.source` like this:

`  s.source       = { :git => "https://github.com/gbumps/react-native-screenguard#release/0.3.6.git", :tag => "#{s.version}" }`

and reinstall the ios again by `cd ios && pod install`.

**Comment by gbumps** [comment]
Hi, i've pushed new commit updated for the iOS. You can reinstall again for iOS by `cd ios && pod install`.

Thanks!

**Comment by gbumps** [comment]
If nothing change, I'm gonna release this version soon for iOS first, while Android still in experimental due to listener event not being triggered when screenshot filter is activated.

**Comment by gbumps** [comment]
This version has been released at version `0.3.6`. Please install and rebuild.

Thanks!

**Comment by Simoon-F** [comment]
Sorry, I 've been quite busy recently. I don't have time to read short messages. I have already tested the bag and found no problems for the time being ~

cool~ 👍
